pinot-broker/src/test/java/org/apache/pinot/broker/broker/BasicAuthAccessControlTest.java

```diff
@@ -101,6 +111,14 @@ public void testDeny() {
     request.setQuerySource(source);
 
     Assert.assertFalse(_accessControl.hasAccess(identity, request));
+
+    Set<String> tableNames = new HashSet<>();
+    tableNames.add("veryImportantStuff");
```

Comment:
is the objective here to test that each of these table names doesn't have access? if so won't the `hasAccess` loop return false when it encounters the first table without access? Just making sure this test does what's intended.

Reply:
Yeah, it is to test that we don't allow access when unless the user has access for all the tables in the query.